python/GangaLHCb/Lib/Applications/GaudiExecUtils.py

```diff
@@ -99,7 +99,6 @@ def prepare_cmake_app(myApp, myVer, myPath='$HOME/cmtuser', myGetpack=None):
     full_path = expandfilename(myPath, True)
     if not path.exists(full_path):
         makedirs(full_path)
-        chdir(full_path)
```

Comment:
@chrisburr Yes I think you are right.

In any case getpack doesn't work for the most part now so I'll put together another PR with lb-cehckout.